api/Infrastructure/Authorization/ProviderAuthorizationHandler.cs

```diff
@@ -20,28 +20,41 @@ public class ProviderAuthorizationHandler : AuthorizationHandler<ProviderAuthori
         protected override Task HandleRequirementAsync(AuthorizationHandlerContext context, ProviderAuthorizationHandler requirement)
         {
             var user = context.User;
+            var httpContext = context.Resource as DefaultHttpContext;
+            var endpoint = httpContext?.GetEndpoint();
+            var actionDescriptor = endpoint?.Metadata.GetMetadata<ControllerActionDescriptor>();
+            var isAuthController = actionDescriptor?.ControllerTypeInfo?.Name == nameof(AuthController);
+            var isUserController = actionDescriptor?.ControllerTypeInfo?.Name == nameof(UsersController);
+
+            if (isUserController && (actionDescriptor.ActionName == nameof(UsersController.RequestAccess) || actionDescriptor.ActionName == nameof(UsersController.GetMyUser)) ||
+            (isAuthController && actionDescriptor.ActionName == nameof(AuthController.UserInfo)))
+            {
+                context.Succeed(requirement);
+                return Task.CompletedTask;
+            }
+
+            if (!user.IsActive())
```

Comment:
@ronaldo-macapobre I added this here to correspond with my changes - however, in current state the session has to expire for updates to this property to change here - we may want to get the user from the db here instead.